HtmlCssSample.html

```diff
@@ -0,0 +1,125 @@
+<!DOCTYPE html>
+<html>
```

Comment:
Add a lang attribute to the **html** element to specify the language of the document.

Reply:
Resolved